addons/mail_plugin/i18n/mail_plugin.pot

```diff
@@ -151,6 +151,13 @@ msgstr ""
 msgid "Search Domain / Email"
 msgstr ""
 
+#. module: mail_plugin
+#. openerp-web
+#: code:addons/mail_plugin/static/src/to_translate/translations_gmail.xml:0
+#, python-format
+msgid "Attachments could not be logged in Odoo because their total size exceeded the allowed maximum."
```

Comment:
As strings are ordered alphabetically, did you put it here randomly or did you re-generate the file ?

Reply:
Oops, we ask me to change the string, but I should have moved it